config/settings/production_aks.yml

```diff
@@ -0,0 +1,39 @@
+gcp_api_key: please_change_me
+publish_api_url: https://api3.publish-teacher-training-courses.service.gov.uk
+publish_url: https://www3.publish-teacher-training-courses.service.gov.uk
+find_url: https://www3.find-postgraduate-teacher-training.service.gov.uk
+
+STATE_CHANGE_SLACK_URL: replace_me
```

Comment:
@JR-G do we need to pass a value to the team here?

Reply:
This is the same as the `production.yml` file so not sure?